plugin/pkg/scheduler/schedulercache/node_info.go

```diff
@@ -90,6 +90,14 @@ func (r *Resource) AddOpaque(name v1.ResourceName, quantity int64) {
 	r.OpaqueIntResources[name] += quantity
 }
 
+func (r *Resource) SetOpaque(name v1.ResourceName, quantity int64) {
+	// Lazily allocate opaque integer resource map.
+	if r.OpaqueIntResources == nil {
+		r.OpaqueIntResources = map[v1.ResourceName]int64{}
+	}
+	r.OpaqueIntResources[name] = quantity
+}
```

Comment:
It would be nice but I don't think so without a larger refactor. The call site within SetNode requires an idempotent operation.